business/checkers/workloads/uncovered_workload_checker.go

```diff
@@ -0,0 +1,52 @@
+package workloads
+
+import (
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/models"
+	security_v1beta "istio.io/client-go/pkg/apis/security/v1beta1"
```

Comment:
nitpick: we usually split the imports in different blocks:
1. native modules
2. dependencies modules
3. kiali modules

Do you mind splitting this import in two? Thanks!